README.md

```diff
@@ -144,7 +158,7 @@ If the feature has multiple scenarios, add them to the same feature file. They w
 
 ## Got Stuck?
 
-Look into our [issues](https://github.com/ttutisani/Xunit.Gherkin.Quick/issues) if your problem was already resolved. Also try [searching StackOverflow](https://stackoverflow.com/search?q=xunit.gherkin.quick).
+Look into our [issues](https://github.com/ttutisani/Xunit.Gherkin.Quick/issues) if your problem was already resolved. Check out the [Samples](https://github.com/ttutisani/Xunit.Gherkin.Quick/tree/master/samples) as well. Also try [searching StackOverflow](https://stackoverflow.com/search?q=xunit.gherkin.quick).
```

Comment:
Actually, you can add a relative link to the samples folder, take a look at the Documentation and Reference section below. It has relative links that always works (in your fork or in my fork does not matter). You should use relative links for things that should work in every fork.

I know you looked at the issues link and did the same, but that should go to a single fork (original repo), Otherwise it does not make sense to take people to other issues page in another fork. So that was an exception.

Reply:
Yeah you are right I usually try to reproduce the style which I find